service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryService.java

```diff
@@ -118,5 +124,14 @@ private String getChildrenForOrder(List<Element<Child>> selectedChildren) {
             ).collect(Collectors.joining(", "))
         ).orElse(null);
     }
+
+    private String getOthersNotified(List<Element<Other>> selectedOthers) {
```

Comment:
could use it's own class like SealedOrderHistoryExtraTitleGenerator so we simplify this service?